library/src/amd_detail/hipblas.cpp

```diff
@@ -3667,8 +3667,13 @@ catch(...)
 }
 
 // swap_batched
-hipblasStatus_t hipblasSswapBatched(
-    hipblasHandle_t handle, int n, float* x[], int incx, float* y[], int incy, int batchCount)
+hipblasStatus_t hipblasSswapBatched(hipblasHandle_t handle,
+                                    int             n,
+                                    float* const    x[],
+                                    int             incx,
+                                    float* const    y[],
+                                    int             incy,
+                                    int             batchCount)
```

Comment:
These changes should be made in nvidia_detail/hipblas.cpp as well. Otherwise looks good.

Reply:
Made similar changes to nvidia_detail/hipblas.cpp